src/ConsentRequiredController.js

```diff
@@ -49,7 +49,7 @@ function (Okta, Util, FormController, FormType, ScopeList) {
         return this.doTransaction(function(transaction) {
           return transaction.cancel();
         }).then(function () {
-          self.settings.get('consent').cancel();
+          self.settings.get('consent.cancel')();
```

Comment:
we should probably check here if the function passed is indeed a function and not undefined or something else
```
var consentCancelFn = self.settings.get('consent.cancel');
_.isFunction(consentCancelFn) {
  consentCancelFn();
}
```